src/components/SiteHeader/index.js

```diff
@@ -269,11 +267,19 @@ const MenuItems = ({ onLogOut, showItems }) => {
   );
 };
 
+const Logo =
+  configuration.logoUrl &&
+  (props => (
+    <a href={configuration.links.home}>
+      <img alt="Logo" src={configuration.logoUrl} style={{ height: configuration.logoHeight }} {...props} />
+    </a>
+  ));
+
```

Comment:
Wouldn't use inline styles here. Instead, make it a styled component.

Reply:
Separated the component and added unit tests.